ocm/component-constructor.yaml

```diff
@@ -5,10 +5,25 @@ components:
     version: "${VERSION}"
     provider:
       name: SAP SE
+    sources:
+      - name: main-source
+        access:
+          type: github
+          repoUrl: github.com/ironcore-dev/gardener-extension-provider-ironcore
+          ref: ${SOURCE_REF}
+          commit: ${SOURCE_COMMIT_SHA}
+        version: ${VERSION}
```

Comment:
for non-release builds you will need a mechanism to ensure the version is unique, e.g. appending commit sha